alacritty/src/logging.rs

```diff
@@ -149,6 +149,16 @@ fn create_log_message(record: &log::Record<'_>, target: &str) -> String {
     message
 }
 
+fn is_allowed_target(level: Level, target: &str) -> bool {
+    use Level::{Error, Warn};
+    use LevelFilter::Trace;
```

Comment:
Please don't put imports inside functions.

Reply:
Changed in latest commit.